pkgs/tools/video/gopro/default.nix

```
@@ -0,0 +1,35 @@
+{ stdenv, fetchFromGitHub
+, ffmpeg
+, imagemagick
+, makeWrapper
+, mplayer
+}:
+
+stdenv.mkDerivation rec {
+  pname = "gopro";
+  version = "1.0";
+
+  src = fetchFromGitHub {
+    owner = "KonradIT";
+    repo = "gopro-linux";
+    rev = version;
+    sha256 = "0sb9vpiadrq8g4ag828h8mvq01fg0306j0wjwkxdmwfqync1128l";
+  };
+
+  nativeBuildInputs = [ makeWrapper ];
+
+  installPhase = ''
```

```suggestion
  installPhase = ''
  runHook preInstall
  
  ...
```

and the same (`postInstall`) at the end. Then it's easier to override.

---

sounds good.